### ISSUE TITLE
Icons: update directory with new icons 

### ISSUE THREAD

**Issue by ekidneyrh** [containers/podman-desktop#7655]
### Is your enhancement related to a problem? Please describe

Currently, it's hard to find what icons are currently in use throughout the app. It would be nice to have them logged somewhere for easy searching.

### Describe the solution you'd like

There was some tracking of used icons in penpot, but it's out of date. 

![image](https://github.com/containers/podman-desktop/assets/87311656/308489cc-fcb1-461b-9021-70fddcbc2322)

I would like to update this and add it to a section of the current Wiki we are building for the Design system. Perhaps making a new category for icon assets.

**Comment by ekidneyrh** [comment]
Assigning this to @joeyyubo - they have access to the containers org but not podman desktop. Will find a way to add them when they are back online.

**Comment by Joeyyubo** [comment]
Some of them (5 items) have errors and cannot be obtained from the code.
- DesktopIcon (Not a valid SVG code)
- SolidKubeIcon(Not a valid SVG code)
- SolidPodIcon(Not a valid SVG code)
- KeyIcon(Error code)
- ServiceIcon (Error code)

**Comment by Joeyyubo** [comment]
[
[Icon for Podman Desktop.zip](https://github.com/user-attachments/files/16735589/Icon.for.Podman.Desktop.zip)
](url)

**Comment by ekidneyrh** [comment]
Thanks for the zip @Joeyyubo! I had a look through it and we need to fix up the svgs - some have a black fill on them that's probably due to an exporting glitch. 

I moved the icons on penpot to here: https://design.penpot.app/#/workspace/ff69ece0-4194-11ec-a943-23056af0be2f/a2efa785-2fc7-80fb-8004-dec816396a8f?page-id=72c6c323-d75c-80f4-8004-dec86cce365e since eventually we want to have a library there :) 

Some icons are missing:

### Containers
When I looked at the svelte file I saw they were font awesome icons:
faCube, faMinusCircle, faPlusCircle from @fortawesome/free-solid-svg-icons

@deboer-tim when I found the icons we were using they were listed under the 'pro' version. 
The closed I can find is this: 

![Screenshot from 2024-08-26 10-10-53](https://github.com/user-attachments/assets/c10ed8d6-126e-4cd4-93a1-81cd37a575e3)

I guess it doesnt matter if we are going to convert them to stroke anyway? 

### Kubernetes engine icon

This is an icon on the Preferences > Kubernetes page

![Screenshot from 2024-08-26 10-30-13](https://github.com/user-attachments/assets/97a8be82-3d22-4c57-b8a6-5dc5ee40cd0e)

It's probably in one of the svelte file. Could you see if you could find it? 

### Action buttons icons

A lot of the action buttons have an icon next to them like this: 

![image](https://github.com/user-attachments/assets/4861f20b-6163-4ef8-bbbf-11d03e9f0ecb)

Could we get those too? Some if not all are Font Awesome icons that can be found on their website https://fontawesome.com/icons. You'll probably have to look though the code to see what exact one is used. It should be at the top of the page files like this:

![Screenshot from 2024-08-26 10-49-06](https://github.com/user-attachments/assets/59a7c841-3075-4868-be89-868e2872f528)

and then you can search the website for the one we use
![image](https://github.com/user-attachments/assets/d0adca40-b19f-4179-80f3-9f77e7a32f0b)

### Others
There's other icons that are used around the app like here on the extensions page:
![Screenshot from 2024-08-26 10-52-34](https://github.com/user-attachments/assets/ae44b49d-b9c8-4753-b56c-2ad3ed826fd1)


You'll have to look are the app and see what other are being used throughout.

For the icons you mentioned above that are missing - do you have where they are located so I can have a look?

**Comment by Joeyyubo** [comment]
Hi Emma. I did some update on Penpot file https://design.penpot.app/#/workspace/ff69ece0-4194-11ec-a943-23056af0be2f/a2efa785-2fc7-80fb-8004-dec816396a8f?page-id=72c6c323-d75c-80f4-8004-dec86cce365e 

Some icons are only pro versions can meet our needs. I agree that we can download the free version and then convert solid svg to stroke.

I did not see where the Kubernetes engine icon file is. 

I went through the app and collected all the icon next pages. I save all icons into a Zip and update to Penpot file already.And there are some I can not find, so I list them in this slides: https://docs.google.com/presentation/d/1JmY1zW5fCN06p4fhrUQ9mlHWjdyz20uc998Q0TzzCiI/edit#slide=id.p 

[Uploading Icon for Podman Desktop.zip…]()

**Comment by ekidneyrh** [comment]
@Joeyyubo Thanks for your work on this! Could you upload the zip of them too? The link to it in ur comment doesn't work. 

I'll have a look and see if I can find the missing icons :)

**Comment by Joeyyubo** [comment]
[Icon for Podman Desktop.zip](https://github.com/user-attachments/files/16961593/Icon.for.Podman.Desktop.zip)

Sorry to put the wrong link.

**Comment by ekidneyrh** [comment]
Had a look at the list of icons you couldnt find.

Desktop icon: this seems to be the podman logo which is ok to leave out of this.

SolidKubeIcon: this is just a solid / filled version of the kube icon.

SolidPodIcon: same goes for this - a solid / filled version of the pod icon.

 I found the key, service, ram, disk image, and kubernetes icons. I updated the folder and Penpot file :) 

[PodmanDesktopIcons.zip](https://github.com/user-attachments/files/17112938/PodmanDesktopIcons.zip)

This ticket can be closed as we have all the icons collected. We will start working on #8043 next.
